### ISSUE TITLE
Input `type="color"` - Disable keyboard interaction with eyedropper button - (2036569946)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5579]
### Module: 
 16 Input &amp; Input Message 

### Violation: 
 Ensure all interactive functionality is operable with the keyboard 

![image](https://user-images.githubusercontent.com/5023024/202759185-348f07b1-4cac-4c85-b9e3-a2361a005e19.png)

### WCAG Reference: 
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) 

### Instance ID: 
 2036569946 

### Severity: 
 8 

### Description: 
 [Issue]
Eyedropper tool (when activated and user needs to select an area to eyedrop) that cannot be navigated to and/or operated with the keyboard alone.

[User Impact]
Keyboard-only users will be prevented from accessing the functionality provided by these controls.

 

### Note: 
 [Suggestion]
Ensure all interactive functionality is operable with the keyboard. The best way to accomplish this is by using appropriate native controls, which come with keyboard functionality built in.

If using a native control is not possible, the control must have tabindex="0" and have JavaScript event listeners that allow the control to be activated when Enter is pressed.

 

### Media Type: 
 Keyboard Accessibility 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This is a native `input` function and cannot be altered.

Additionally the same functionality is possible via alternative means via keyboard, where users can select a color with the arrow keys, or change the HEX/HSV/RGB value.

Closing as no code change can be made.
